metrics/bigfiles_high_water_mark

```diff
@@ -1 +1 @@
-897
+967
```

Comment:
After this, `table_validator.py` and `types/stubs/logging/__init__.pyi` are off of the list.  The MySQL additions to `setup.py` and `itest` are still hanging around, though - I'm proposing to accept that increase for now and take on refactoring those two pieces as part of their respective backlog items.

```
350: itest
317: records_mover/records/schema/field/__init__.py
300: setup.py

Reduce total number of bigfiles violations to 897 or below!

Tasks: TOP => bigfiles
(See full trace by running task with --trace)
Existing violations: 897
Found 967 bigfiles violations
```